src/util/images.tsx

```diff
@@ -48,9 +48,17 @@ export const localLxdToRemoteImage = (image: LxdImage): RemoteImage => {
     arch: image.architecture === "x86_64" ? "amd64" : image.architecture,
     os: image.properties?.os ?? "",
     created_at: new Date(image.uploaded_at).getTime(),
-    release: image.properties?.release ?? "",
-    server: LOCAL_IMAGE,
+    release:
+      image.properties?.os === "ubuntu"
+        ? (image.properties.version ?? "")
+        : (image.properties?.release ?? ""),
```

Comment:
```suggestion
```
Why is this needed?
There is similar special handling of Ubuntu in the image selector component. The goal is to have it read nicely "Ubuntu xx.xx codename" as people are used to that ordering.

Reply:
This is added so that we can populate the "Release" column within the modal. The ternary condition is needed because for an `LxdImage` returned from the server, the data is not organised exactly the same as a `RemoteImage`. The field `image.properties.version` in this case for a cached `ubuntu` image aligns with the "Release" column content populated with the remote image data.

Reply:
Ah I see, this is handling the cached entries and transforming them into the same format as the ones we fetch from image servers.

Reply:
Still suspicious if we should have the special handling of Ubuntu here and in the modal.

Reply:
Suggesting to make this funciton more straight forward:
```
export const localLxdToRemoteImage = (image: LxdImage): RemoteImage => {
  return {
    aliases: image.update_source?.alias ?? image.aliases?.[0]?.name ?? "",
    fingerprint: image.fingerprint,
    arch: image.architecture === "x86_64" ? "amd64" : image.architecture,
    os: image.properties?.os ?? "",
    created_at: new Date(image.uploaded_at).getTime(),
    release: image.properties?.release ?? "",
    release_title: image.properties?.version ?? "",
    server: LOCAL_IMAGE,
    type: image.type,
  };
};
```
And keep the special handling of ubuntu in the ImageSelector.
If we pass the os here in this function through a converter, that ensures the first character is uppercase, it all should work well.

Reply:
yeah agreed, will implement that. I would keep the `cached` field though since we want to differentiate later on whether a local image is a cached remote image or uploaded image.